SPECS-EXTENDED/rubygem-abrt/rubygem-abrt.spec

```
@@ -1,29 +1,24 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-# Generated from abrt-0.0.2.gem by gem2rpm -*- rpm-spec -*-
 %global gem_name abrt
-
-Name: rubygem-%{gem_name}
-Version: 0.4.0
-Release: 2%{?dist}
-Summary: ABRT support for Ruby
-License: MIT
-URL: http://github.com/voxik/abrt-ruby
-Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-# git clone https://github.com/voxik/abrt-ruby.git && cd abrt-ruby
-# git archive -v -o abrt-0.4.0-spec.tar.gz v0.4.0 spec/
-Source1: %{gem_name}-%{version}-spec.tar.gz
-Requires: libreport-filesystem
+Summary:       ABRT support for Ruby
+Name:          rubygem-%{gem_name}
+Version:       0.4.0
+Release:       2%{?dist}
+License:       MIT
+URL:           https://github.com/voxik/abrt-ruby
+Vendor:        Microsoft Corporation
+Distribution:  Mariner
+#Source0:      https://github.com/voxik/abrt-ruby/archive/refs/tags/v%{version}.tar.gz
+Source0:       %{gem_name}-ruby-%{version}.tar.gz
```

Preferred:

```suggestion
Source0:      https://github.com/voxik/abrt-ruby/archive/refs/tags/v%{version}.tar.gz#/%{gem_name}-ruby-%{version}.tar.gz
```

May I ask you to update other modified specs in the same way?